api_contracts/contracts/search/AdvancedSearch.md

```diff
@@ -108,7 +132,9 @@ This endpoint returns a list of aggregated award amounts grouped by time period
 + recipient_locations (optional, array[LocationObject])
 + recipient_type_names: `category_business` (optional, array[string])
     See options at https://github.com/fedspendingtransparency/usaspending-api/wiki/Recipient-Business-Types
-+ award_type_codes: A, B, C, D (optional, array[string])
++ award_type_codes: 02, 03, 04, 05, 06, 07, 08, 09, 10, 11, A, B, C, D, E, F, G, S, T, IDV_A, IDV_B, IDV_C, IDV_D, IDV_E (optional, array[string])
```

Comment:
Since these are strings, do they need backticks on them?
```
`02`,`03`,`04`,`05`,`06`,`07`,`08`,`09`,`10`,`11`,`A`,`B`,`C`,`D`,`E`,`F`,`G`,`S`,`T`,`IDV_A`,`IDV_B`,`IDV_C`,`IDV_D`,`IDV_E`,
```

Reply:
yep, just added them!